--- a/drivers/mmc/card/emmc_partitions.c
+++ b/drivers/mmc/card/emmc_partitions.c
@@ -595,6 +595,10 @@ int aml_emmc_partition_ops (struct mmc_c
         // printk("[%s] %s is not eMMC!\n", __FUNCTION__, disk->disk_name);
         return 0;
     }
+
+    // if (!of_find_node_by_path("/partitions"))
+        return 0;
+
     store_device = host->storage_flag;
     pt_fmt = kmalloc(sizeof(struct mmc_partitions_fmt), GFP_KERNEL);
     if(pt_fmt == NULL){
